src/rai_bench/rai_bench/examples/tool_calling_agent_test_bench.py

```diff
@@ -0,0 +1,167 @@
+
+    tasks: Sequence[ToolCallingAgentTask] = [
+        GetROS2RGBCameraTask(logger=bench_logger),
+        GetROS2TopicsTask(logger=bench_logger),
+        GetROS2DepthCameraTask(logger=bench_logger),
+        GetAllROS2RGBCamerasTask(logger=bench_logger),
+        GetROS2TopicsTask2(logger=bench_logger),
+        GetROS2MessageTask(logger=bench_logger),
+        MoveToPointTask(
+            logger=bench_logger, args={"x": 1.0, "y": 2.0, "z": 3.0, "task": "grab"}
+        ),
+        MoveToPointTask(
+            logger=bench_logger, args={"x": 1.2, "y": 2.3, "z": 3.4, "task": "drop"}
+        ),
+        GetObjectPositionsTask(
+            logger=bench_logger,
+            objects={
+                "carrot": [{"x": 1.0, "y": 2.0, "z": 3.0}],
+                "apple": [{"x": 4.0, "y": 5.0, "z": 6.0}],
+                "banana": [
+                    {"x": 7.0, "y": 8.0, "z": 9.0},
+                    {"x": 10.0, "y": 11.0, "z": 12.0},
+                ],
+            },
+        ),
+        GrabExistingObjectTask(
+            logger=bench_logger,
+            object_to_grab="banana",
+            objects={
+                "banana": [{"x": 7.0, "y": 8.0, "z": 9.0}],
+                "apple": [
+                    {"x": 4.0, "y": 5.0, "z": 6.0},
+                    {"x": 10.0, "y": 11.0, "z": 12.0},
+                ],
+            },
+        ),
+        GrabNotExistingObjectTask(
+            logger=bench_logger,
+            object_to_grab="apple",
+            objects={
+                "banana": [{"x": 7.0, "y": 8.0, "z": 9.0}],
+                "cube": [
+                    {"x": 4.0, "y": 5.0, "z": 6.0},
+                    {"x": 10.0, "y": 11.0, "z": 12.0},
+                ],
+            },
+        ),
+        MoveExistingObjectLeftTask(
+            logger=bench_logger,
+            object_to_grab="banana",
+            objects={
+                "banana": [{"x": 7.0, "y": 8.0, "z": 9.0}],
+                "apple": [
+                    {"x": 4.0, "y": 5.0, "z": 6.0},
+                    {"x": 10.0, "y": 11.0, "z": 12.0},
+                ],
+            },
+        ),
+        MoveExistingObjectFrontTask(
+            logger=bench_logger,
+            object_to_grab="banana",
+            objects={
+                "banana": [{"x": 7.0, "y": 8.0, "z": 9.0}],
+                "apple": [
+                    {"x": 4.0, "y": 5.0, "z": 6.0},
+                    {"x": 10.0, "y": 11.0, "z": 12.0},
+                ],
+            },
+        ),
+        SwapObjectsTask(
+            logger=bench_logger,
+            objects={
+                "banana": [{"x": 1.0, "y": 2.0, "z": 3.0}],
+                "apple": [{"x": 4.0, "y": 5.0, "z": 6.0}],
+            },
+            objects_to_swap=["banana", "apple"],
+        ),
+    ]
+    benchmark = ToolCallingAgentBenchmark(
+        tasks=tasks, logger=bench_logger, results_filename=results_filename
+    )
+
+    for _, task in enumerate(tasks):
```

Comment:
what's the point of enumerate if you skip it `_`?
```suggestion
    for task in tasks:
```